<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1020</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Aurora - OpSys2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QwtPlot" name="ctfPlot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>650</y>
      <width>731</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>361</width>
      <height>361</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="ctfRealLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>341</width>
       <height>341</height>
      </rect>
     </property>
     <property name="text">
      <string>Real CTF</string>
     </property>
    </widget>
   </widget>
   <widget class="QDoubleSpinBox" name="ctfAngleSpinBox">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>750</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>-1000000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_57">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>750</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Angle</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>70</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>mrad</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="apertureSpinBox">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>40</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.400000000000000</double>
    </property>
    <property name="value">
     <double>100.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>40</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Aperture:</string>
    </property>
   </widget>
   <widget class="QLabel" name="wavelengthLabel_1">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Wavelength:</string>
    </property>
   </widget>
   <widget class="QLabel" name="pmLabel">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>pm</string>
    </property>
   </widget>
   <widget class="QLabel" name="accelerationVoltageLabel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Energy:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="energySpinBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>10000000.000000000000000</double>
    </property>
    <property name="value">
     <double>100.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="wavelengthLabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>?</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="kevLabel">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>130</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>eV</string>
    </property>
   </widget>
   <widget class="QLabel" name="accelerationVoltageLabel_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Maximal Frequency:</string>
    </property>
   </widget>
   <widget class="QLabel" name="kevLabel_2">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>10</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>1/nm</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="maxFrequencySpinBox">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>10</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>10000000.000000000000000</double>
    </property>
    <property name="value">
     <double>50.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="wavelengthLabel_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>160</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>High Tension Ripple:</string>
    </property>
   </widget>
   <widget class="QLabel" name="wavelengthLabel_4">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>250</y>
      <width>276</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit Wave</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="accelerationVoltageLabel_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Cc:</string>
    </property>
   </widget>
   <widget class="QLabel" name="accelerationVoltageLabel_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Semi-Angle:</string>
    </property>
   </widget>
   <widget class="QLabel" name="accelerationVoltageLabel_5">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>130</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Energy Spread:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="semiAngleSpinBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>70</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="maximum">
     <double>10000000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="pmLabel_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>100</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Âµm</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="cCSpinBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>100</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>10000000.000000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="energySpreadSpinBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>130</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="maximum">
     <double>10000000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="focusSpreadLabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>220</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>?</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="wavelengthLabel_10">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>190</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Lens Current Instability:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="htRippleSpinBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>160</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="maximum">
     <double>10000000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="currentInstabilitySpinBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>190</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="maximum">
     <double>10000000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="wavelengthLabel_11">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>220</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Focus Spread:</string>
    </property>
   </widget>
   <widget class="QLabel" name="pointResolutionLabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>250</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>?</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="kevLabel_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>keV</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>160</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>ppm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_31">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>190</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>ppm</string>
    </property>
   </widget>
   <widget class="QLabel" name="kevLabel_4">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>40</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>mrad</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_32">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>220</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>nm</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_6">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>280</y>
      <width>361</width>
      <height>361</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="ctfImagLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>341</width>
       <height>341</height>
      </rect>
     </property>
     <property name="text">
      <string>Imag CTF</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="ctfWpoaImagCheckBox">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>850</y>
      <width>90</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>imag(CTF)</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="ctfWpoaRealCheckBox">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>820</y>
      <width>90</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>real(CTF)</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="envelopeCheckBox">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>790</y>
      <width>90</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Envelope</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_36">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>750</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>deg</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="dataPointsSpinBox">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>890</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="ctfRealCheckBox">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>880</y>
      <width>90</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>real(CTF)</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="ctfImagCheckBox">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>910</y>
      <width>90</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>imag(CTF)</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>840</y>
      <width>57</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>WPOA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_68">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>250</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>1 / nm</string>
    </property>
   </widget>
   <widget class="Aurora::QViewIntern" name="view" native="true">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>10</y>
      <width>532</width>
      <height>835</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frame_7">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>280</y>
      <width>276</width>
      <height>276</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="exitWaveLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>256</width>
       <height>256</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit Wave</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_8">
    <property name="geometry">
     <rect>
      <x>1021</x>
      <y>280</y>
      <width>276</width>
      <height>276</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="phaseShiftLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>256</width>
       <height>256</height>
      </rect>
     </property>
     <property name="text">
      <string>Phaseplate</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="loadExitWaveButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>560</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Exit Wave</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>560</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Phaseplate</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>361</width>
      <height>261</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Coefficient</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Value</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Unit</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Angle</string>
     </property>
    </column>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>70</y>
      <width>301</width>
      <height>104</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="pmLabel_6">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>nm</string>
     </property>
    </widget>
    <widget class="QLabel" name="scherzerDefocusLabel">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="scherzerButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Scherzer Defocus</string>
     </property>
    </widget>
    <widget class="QPushButton" name="focusButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Focus</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="scherzerOrderSpinBox">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QPushButton" name="lichteButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Lichte Defocus</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="wavelengthLabel_12">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>250</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Point Resolution:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveSettingsButton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>180</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Settings</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadSettingsButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>180</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Settings</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QWidget</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Aurora::QViewIntern</class>
   <extends>QWidget</extends>
   <header>QtSupport/QView.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>energySpinBox</tabstop>
  <tabstop>semiAngleSpinBox</tabstop>
  <tabstop>cCSpinBox</tabstop>
  <tabstop>energySpreadSpinBox</tabstop>
  <tabstop>htRippleSpinBox</tabstop>
  <tabstop>currentInstabilitySpinBox</tabstop>
  <tabstop>maxFrequencySpinBox</tabstop>
  <tabstop>apertureSpinBox</tabstop>
  <tabstop>focusButton</tabstop>
  <tabstop>scherzerButton</tabstop>
  <tabstop>scherzerOrderSpinBox</tabstop>
  <tabstop>lichteButton</tabstop>
  <tabstop>ctfAngleSpinBox</tabstop>
  <tabstop>envelopeCheckBox</tabstop>
  <tabstop>ctfWpoaRealCheckBox</tabstop>
  <tabstop>ctfWpoaImagCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
